wallet/androidTest/de/schildbach/wallet/database/DatabaseMigrationTest.kt

```diff
@@ -54,7 +74,37 @@ open class DatabaseMigrationTest {
     @Throws(IOException::class)
     fun migrateAll() {
         // Create db and fill with data
-        testHelper.createDatabase(TEST_DB_NAME, 8).apply {
+        testHelper.createDatabase(TEST_DB_NAME, 1).apply {
```

Comment:
Similar to the other PR and perhaps unnecessary, this test shows that starting from version 1, that no data is migrated to version 12.